### PR TITLE
fix total kafka consumer lag

### PR DESCRIPTION
**Pull request by automatedops**

### CODE REVIEW

**Comment by shortdudey123** [comment]
:+1:

File: src/collectors/kafka_consumer_lag/test/testkafka_consumer_lag.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 class TestKafkaConsumerLagCollector(CollectorTestCase):
     def setUp(self):
         config = get_collector_config('KafkaConsumerLagCollector', {
-            'consumer_groups': ['test_group']
+            'consumer_groups': ['stage_nginx_access']
```
Comment (gregorygtseng) [inline src/collectors/kafka_consumer_lag/test/testkafka_consumer_lag.py:18]: why is this needed

Comment (automatedops) [inline src/collectors/kafka_consumer_lag/test/testkafka_consumer_lag.py:18]: Because `stage_nginx_access` should match results.
